sos/__init__.py

```diff
@@ -34,12 +34,6 @@ def _default(msg):
 
 _sos = _default
 
-# py3 < 3.6 compat
-try:
-    ModuleNotFoundError
-except NameError:
-    ModuleNotFoundError = ImportError
-
```

Comment:
We may need to leave this as is, and disable the check here while we're still supporting python 3.6 for both bionic and centos 8, till later in the year

Reply:
It's needed for < Python 3.6. It should be OK for 3.6 or later; I thought the minimum version is 3.6?